### DISCUSSION TITLE
IntelliJ plugin *very* inconsistently allows me to apply suggested completions in the code by using the designated shortcut

### DISCUSSION THREAD

**Discussion by StylianosGakis** [community/community#76912] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Using GitHub Copilot IntelliJ IDE Plugin version: 1.4.2.3864
I got the shortcut in "Keymap > Plugins > GitHub Copilot > Copilot: Apply Completions to Editor" set to `tab` and to `option + tab`
<img width="1166" alt="image" src="https://github.com/community/community/assets/44558292/00f4ddcf-a5b4-40ee-a556-5cb3ab703f1c">
But when actually trying to use it, it is incredibly inconsistent. Sometimes just pressing tab works by itself. But most of the times, it simply adds a `tab` to the content, does not autocomplete copilot at all, despite the fact that I see the suggestion right in front of my eyes. More often than not this way, I need to then just type out myself what the suggestion is, which is an incredibly frustrating experience, to the point that I'd honestly rather not get the suggestion in the first place.
The worst part is that it does work sometimes, but I can't understand which times, it's not consistent at all. Which means that often I will press the button assuming it will work, but it does not. If it *never* worked at least I would've known there's something completely wrong setup.

I have also tried removing the tab shortcut even from the actual tab action itself, so pressing tab does not even add the spaces, but I still experience the same thing, pressing tab then just does nothing, and the prompt is still there right in front of me, unable to be applied.

If I open the GitHub Copilot window and I see all the suggested texts, and I manually click on the "Accept Solution" button then that does work as well, but of course that's not a good developer experience, it's much slower than entering the text myself again.

**Comment by chomik-pawel** [comment]
Hello there, I wanted to start a new issue but I found your topic and I have the same experience. Apply completions is inconsistent and it's very very annoying. I noticed that when it works you can find that option in the _Actions_ but when it's in the "bugged" state then it's not there. So it's not matter of shortcut not being triggered but rather option is unavailable at time but inline suggestion is there... @Support please help us
